core/src/test/scala/io/chrisdavenport/epimetheus/mules/CacheLookupCounterSpec.scala

```diff
@@ -46,18 +47,22 @@ class CacheLookupCounterSpec extends mutable.Specification {
         _ <- cache2.insert(3, 0D)
         _ <- newCache2.lookup(3)
         out <- cr.write004
-      } yield out
+      } yield dropCreatedLines(out, 3)
 
       val expected = 
       """# HELP mules_cache_lookup_total Cache Lookup Status Counter.
         |# TYPE mules_cache_lookup_total counter
         |mules_cache_lookup_total{cache_name="foo",status="miss",} 1.0
         |mules_cache_lookup_total{cache_name="bar",status="hit",} 1.0
         |mules_cache_lookup_total{cache_name="foo",status="hit",} 1.0
-        |""".stripMargin
+        |# HELP mules_cache_lookup_created Cache Lookup Status Counter.
+        |# TYPE mules_cache_lookup_created gauge""".stripMargin
```

Comment:
With the new prometheus version we also get a `_created` metric.

The timestamp makes these tests a bit more difficult. At the moment I am dropping those lines, may need to look if we can do something a bit cleaner.

Here is an example of the full output:

```
# HELP mules_cache_lookup_total Cache Lookup Status Counter.
# TYPE mules_cache_lookup_total counter
mules_cache_lookup_total{cache_name="foo",status="miss",} 1.0
mules_cache_lookup_total{cache_name="bar",status="hit",} 1.0
mules_cache_lookup_total{cache_name="foo",status="hit",} 1.0
# HELP mules_cache_lookup_created Cache Lookup Status Counter.
# TYPE mules_cache_lookup_created gauge
mules_cache_lookup_created{cache_name="foo",status="miss",} 1.632906700497E9
mules_cache_lookup_created{cache_name="bar",status="hit",} 1.632906700498E9
mules_cache_lookup_created{cache_name="foo",status="hit",} 1.632906700493E9
```